test/val/val_id_test.cpp

```diff
@@ -2651,7 +2651,7 @@ TEST_F(ValidateIdWithMessage, OpCopyMemoryVoidTarget) {
 %2 = OpTypeInt 32 0
 %3 = OpTypePointer Uniform %1
 %4 = OpTypePointer Uniform %2
-%5 = OpTypeFunction %1 %2 %3
+%5 = OpTypeFunction %1 %3 %4
```

Comment:
This was incorrect, but the order of checks previously exited before hitting it.